tests/e2e_test.go

```diff
@@ -91,24 +116,14 @@ var _ = ginkgo.BeforeSuite(func() {
 		fundedKey,
 	)
 
-	var ctx context.Context
-	ctx, cancelDecider = context.WithCancel(context.Background())
-	// we'll call cancelDecider in AfterSuite, but also call it if this
-	// process is killed, because AfterSuite won't always run then:
-	signalChan := make(chan os.Signal, 2)
-	signal.Notify(signalChan, os.Interrupt, syscall.SIGTERM)
-	go func() {
-		<-signalChan
-		cancelDecider()
-	}()
 	decider = exec.CommandContext(ctx, "./tests/cmd/decider/decider")
 	decider.Start()
-	go func() { // panic if the decider exits abnormally
+	go func() {
```

Comment:
I know this was the existing behavior, but it's kind of odd that the decider persists across all tests, while the aggregator and relayer services are spun up and torn down by each test as needed. I think we should do the same for the decider.

As far as the E2E tests go, I don't think the existing relayer E2E tests need to run the decider service at all. We should define a test that explicitly runs the no-op test decider, though.

Reply:
(We can defer this to a separate change, I know it's not exactly in scope of the changes you've made)

Reply:
Filed #452